shared/scripts/virtio_console_guest.py

```diff
@@ -24,6 +24,8 @@
 import six
 from six.moves import xrange
 from six.moves import input
+import platform
```

Comment:
please keep your import statements ordered.

Reply:
Thanks and update.